### PR TITLE
abbr: add pt_BR translation

### PR DESCRIPTION
**Pull request by CasperBraske**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pt_BR/linux/abbr.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# abbr
+
+> Gerencie abreviaçoes para fish-shell.
+> Palavras definidas pelo usuário são substituídas por frases longas assim que são digitadas.
+> Mais informações: <https://fishshell.com/docs/current/cmds/abbr.html>.
+
+- Adicione uma nova abreviação:
+
+`abbr --add {{abbreviation_name}} {{command}} {{command_arguments}}`
```
Comment (navarroaxel) [inline pages.pt_BR/linux/abbr.md:9]: Please translate the placeholders for get this easier for non-English speakers, e.g. `command` to `comando`.

**Comment by CasperBraske** [comment]
Done.

**Comment by CasperBraske** [comment]
I'm using the browser UI to add translations and _for some reason_ Github added my last commit to this PR. I apologise.

**Review by EmilySeville7cfg — APPROVED** [review]
LGTM ;)

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**